plugins/tests/pod/test_pod.py

```diff
@@ -81,6 +87,57 @@ def simple_pod_task(i: int):
     assert config["primary_container_name"] == "a container"
 
 
+def test_pod_task():
```

Comment:
Not sure why github renders like this This is exactly the same test as before with the underscores changed to camelcase.